pkg/popeye.go

```diff
@@ -118,7 +118,8 @@ func (p *Popeye) dump(printHeader bool) error {
 		}
 		fmt.Fprintf(p.outputTarget, "%v\n", res)
 	case report.PrometheusFormat:
-		pusher := p.builder.ToPrometheus(p.flags.PushGatewayAddress)
+		pusher := p.builder.ToPrometheus(p.flags.PushGatewayAddress,
+			p.client.ActiveCluster(), p.client.ActiveNamespace())
 		err := pusher.Add()
 		if err != nil {
```

Comment:
@eminugurkenar Why is this errors commented out?

Reply:
I accually just followed same thing that this popeye.go , for instance for json and yaml it is also commented out. Removed it now it just returns the error , as caller of dump function handles the error ,it should be fine.